### PR TITLE
Changes to enable tests in k8s

### PR DESCRIPTION
**Pull request by shanemcd**
Depends-On: https://github.com/ansible/zuul-jobs/pull/30

I've been poking around on getting our tests to run in a more reproducible fashion using Kubernetes as the main execution environment. After this merges we can merge https://github.com/ansible/zuul-jobs/pull/30 (when it's done). *Note* Merging this PR won't change or break anything in our current environment.

### CODE REVIEW

**Comment by shanemcd** [comment]
recheck

File: tools/docker-compose/start_tests.sh [hunk 1]
```diff
@@ -1,12 +1,6 @@
 #!/bin/bash
 set +x
 
-if [ `id -u` -ge 500 ] || [ -z "${CURRENT_UID}" ]; then
```
Comment (shanemcd) [inline tools/docker-compose/start_tests.sh:4]: Drive-by cleanup. This hasn't been necessary since https://github.com/ansible/awx/pull/3870 merged

File: Makefile [hunk 2]
```diff
@@ -496,6 +496,10 @@ ui-devel: $(UI_DEPS_FLAG_FILE)
 ui-test: $(UI_DEPS_FLAG_FILE)
 	$(NPM_BIN) --prefix awx/ui run test
 
+ui-lint: $(UI_DEPS_FLAG_FILE)
```
Comment (shanemcd) [inline Makefile:500]: I moved this out of tox into a Make target so that I can use it elsewhere.

File: tools/docker-compose/Dockerfile [hunk 3]
```diff
@@ -6,14 +6,20 @@ RUN yum -y update && yum -y install epel-release
 
 # sync with installer/roles/image_build/templates/Dockerfile.j2
 RUN yum -y install acl \
+  alsa-lib \
```
Comment (shanemcd) [inline tools/docker-compose/Dockerfile:9]: Adding our UI deps to our devel image to simplify our environments. Since we take advantage of caching, this shouldn't be too painful. Refactoring things out into smaller components should be a separate task in the future.

**Comment by shanemcd** [comment]
@matburt I think this side of things is ready. Once this merges I can finish up https://github.com/ansible/zuul-jobs/pull/30. Merging this in won't break anything in the current setup.

**Comment by shanemcd** [comment]
recheck

**Comment by shanemcd** [comment]
@matburt @ryanpetrello Can I get some help taking care of the test that's failing because it expects rabbitmq to be up?

**Comment by shanemcd** [comment]
recheck

**Review by matburt — APPROVED** [review]

**Comment by shanemcd** [comment]
regate

**Comment by shanemcd** [comment]
regate
